packages/shared/src/components/sidebar/MyFeedButton.tsx

```diff
@@ -80,9 +48,15 @@ const UnfilteredMyFeedButton = ({
   const { trackEvent } = useContext(AnalyticsContext);
 
   const buttonCopy = getFeatureValue(Features.MyFeedButtonCopy, flags);
-  const buttonColor = getFeatureValue(Features.MyFeedButtonColor, flags);
+  const buttonColor = getThemeColor(
+    getFeatureValue(Features.MyFeedButtonColor, flags),
+    'primary',
```

Comment:
Don't we want to use the default value of the feature instead of falling back to `primary`? I think it makes more sense